SECURITY.md

```diff
@@ -1,30 +1,30 @@
 # Security Policy
 
-Reference client for NEAR is held to highest security standard.
-This document defines the policy how to report vulnerabilities and receive updates when security patches are released.
+Reference client for NEAR is held to the highest security standard.
+This document defines the policy on how to report vulnerabilities and receive updates when security patches are released.
 
 If you have any suggestions or comments about the security policy, please email the [NEAR Security Team](mailto:security@near.org) at security@near.org
 
 ## Reporting a vulnerability
 
-All security issues and questions should be reported by sending email to [NEAR Security Team](mailto:security@near.org) at security@near.org.
-This will be acknowledged within 24 hours by the NEAR Security Team and kick of review process.
-You will receive a more detailed response to the email within 72 hours indicating perceived severity and the next steps in handling your report.
+All security issues and questions should be reported via email to the [NEAR Security Team](mailto:security@near.org) at security@near.org.
```

Comment:
‘by sending an email’ would also be correct.

Reply:
Agreed — had a dilemma but picked `via` as it was more concise!